DSharpPlus.Lavalink/Entities/LavalinkDispatch.cs

```diff
@@ -25,18 +25,21 @@
 
 namespace DSharpPlus.Lavalink.Entities
 {
-    internal sealed class VoiceDispatch
+    public class LavalinkDispatch
```

Comment:
What was the reason behind this visibility change?

Reply:
Why is it no longer sealed too?